modules/contentbox/models/system/CBHelper.cfc

```diff
@@ -201,7 +201,7 @@ component accessors="true" singleton threadSafe{
 	function siteBaseURL(){
 		return replacenocase( 
 			getRequestContext().buildLink( linkTo='', ssl=getRequestContext().isSSL() ), 
-			"index.cfm", 
+			"index.cfm/", 

```

Comment:
because everyone is complaining that we are having // everywhere in the base href and links.